tests/client/test_version.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=invalid-name
+
+import re
+
+from tests import BASE_URL, MASTER_KEY
+from meilisearch.version import *
```

Comment:
If you move the version to `__init__.py` this would change to:

```py
from meilisearch import __version__
```

Also, `import *` is generally discouraged in Python.